src/contexts/UserContext.tsx

```diff
@@ -25,14 +25,10 @@ export const UserProvider = ({ children }: Props) => {
         setCurrentUser
     }
 
+    console.log(currentUser)
```

Comment:
Should we exclude console.log() statements on the main branch?

Reply:
Yea, definitely. Totally forgot to remove that after I was done making edits.